### ISSUE TITLE
Feature Request: Aircraft type

### ISSUE THREAD

**Issue by Marsfan** [antirez/dump1090#66]
Add identification of aircraft type.

**Comment by yuvadm** [comment]
@Marsfan Are you sure this is part of the ADS-B spec? I'm not sure at all.

**Comment by Marsfan** [comment]
I'm not sure. I just know that another application I use (adsb#/Adsbscope) can tell me what plane type it is (b763, a321, etc).

**Comment by hhrhhr** [comment]
Adsbscope use a 3rd-party db (adsbscope\extra\icao24plus.txt)

**Comment by marutan** [comment]
I thought I'd drop a line to say that I've implemented this, reading the icao24plus.txt file from adsb scope. I just need to tidy the patch up slightly (and then create a repo on github to pull from)

**Comment by SeanMaday** [comment]
Is it possible to have this field passed to data.json?

**Comment by marutan** [comment]
Yes, infact I only output this on the JSON, I didn't want to change the format of anything where dump1090 was acting as 'headless'.

I realised only later that I'd done this on the MalcommRobb fork of dump1090, you can see the pull request and a screenshot over here.

https://github.com/MalcolmRobb/dump1090/pull/70

**Comment by quentin9696** [comment]
Hello, were can I find an updated file of icao24.txt ? 

Is this file is up to date http://www.tech-software.net/RTL1090/adsb_scope/pc_software/adsbscope/26/extra/icao24plus.txt ? 

Regards,
